config/sync/pineapple.settings.yml

```diff
@@ -0,0 +1,11 @@
+features:
+  node_user_picture: true
+  comment_user_picture: true
+  comment_user_verification: 1
+  favicon: 1
+logo:
+  use_default: 1
+favicon:
+  use_default: 0
+  path: 'public://easy-recipe-delight-favicon.png'
```

Comment:
Favicons should be .ico not png, please get your favicon in the correct format

Reply:
done